File: DTO/Models/Wine.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DTO.Models
{
    public class Wine
    {
        public int Id { get; set; }
        public string WineName { get; set; }
        public string Description { get; set; }
        public int Year { get; set; }
        public bool Disabled { get; set; }
        public List<Category> Category { get; set; }
        public List<Price> Prices { get; set; }
        public List<Comment> Comments { get; set; }
    }
}

File: Angular-Project/WineSellingProject/src/app/Models/Wine.ts
import { Category } from "./Category";
import { Price } from "./Price";

export class Wine {
    id: number;
    wineName: string;
    disabled: boolean;
    prices: Price[];
    year: number;
    description: string;
    category: Category[];

    constructor(id: number,wineName: string, disabled: boolean, prices: Price[], year, description: string, category: Category[]) {
        this.id = id;
        this.disabled = disabled;
        this.prices = prices;
        this.year = year;
        this.wineName = wineName;
        this.description = description;
        this.category = category;
    }
}

